backend/webapi/Features/Parties/BcProvider.cs

```diff
@@ -0,0 +1,34 @@
+namespace Pidp.Features.Parties;
+
+using FluentValidation;
+using Microsoft.EntityFrameworkCore;
+
+public class BcProvider
```

Comment:
I would call this `BCProviderDetails`, personally.
Note that the naming conventions for C# say that two letter acronyms should have both letters capitalized:
https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions#capitalization-rules-for-identifiers

Reply:
I've made the change. I have been ignoring this exception to the casing convention for a long time, roughly since the Id (vs. ID) casing became popular.